examples/js/loaders/DRACOLoader.js

```diff
@@ -0,0 +1,202 @@
+// Copyright 2016 The Draco Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+THREE.DRACOLoader = function(manager) {
+    this.manager = (manager !== undefined) ? manager :
+        THREE.DefaultLoadingManager;
+    this.materials = null;
+};
+
+const DracoModule = Module;
```

Comment:
It looks like this library is built with Emscripten, which is where the global `Module` variable is coming from.  Emscripten supports a couple arguments at compile time which lets you provide proper namespaced modules for compiled code.  

The arguments you want would be something like `-s MODULARIZE=1 -s EXPORT_NAME="'DRACO'"` which will create the module definition in `window.DRACO` instead of `window.Module`.